extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java

```diff
@@ -169,6 +168,10 @@ private String guessDriver(Optional<JdbcDataSourceBuildItem> jdbcDataSource) {
         return logCleanUps;
     }
 
+    private String getStoreClassSimpleName(String clazz) throws ClassNotFoundException {
+        return Class.forName(clazz).getSimpleName();
```

Comment:
Do we need to load the class here? We could use something like `String.lastIndexOf(".")`, or?

Reply:
Yes, thought about that but I assumed it is okay since we are at build time?

Reply:
Yes, it's probably ok. But still, we load a class just to get the simple name...

Reply:
Okay, gotcha. I have updated the code to take into account this feedback.